languages/cpp/README.md

```diff
@@ -51,7 +52,7 @@ ProjectResponse projectResponseGet = bitwardenClient.getProject(projectId);
 
 ```c++
 boost::uuids::uuid projectId = boost::uuids::string_generator()(projectResponseCreate.get_id());
-ProjectResponse projectResponseUpdate = bitwardenClient.updateProject(projectId, organizationUuid, "TestProjectUpdated");
+ProjectResponse projectResponseUpdate = bitwardenClient.updateProject(organizationUuid, projectId, "TestProjectUpdated");
 ```
 
 ### Delete projects
```

Comment:
In delete projects we delete secrets. Can we fix this too ?

Reply:
Good catch!

Reply:
Fixed in 7ab21345d0d91c7dd4a152ef7b33da507f2bae6b.